cdc/log.cc

```diff
@@ -1289,7 +1290,11 @@ class transformer final {
         auto partition_slice = query::partition_slice(std::move(bounds), std::move(static_columns), std::move(regular_columns), std::move(opts));
         auto command = ::make_lw_shared<query::read_command>(_schema->id(), _schema->version(), partition_slice, row_limit);
 
-        return _ctx._proxy.query(_schema, std::move(command), std::move(partition_ranges), cl, service::storage_proxy::coordinator_query_options(default_timeout(), empty_service_permit(), client_state)).then(
+        const auto select_cl = write_cl >= db::consistency_level::QUORUM
+                ? db::consistency_level::QUORUM
+                : db::consistency_level::ONE;
```

Comment:
We can't use `>=` for comparing the strength of consistency levels. For example, it won't work for LOCAL_ONE - it has the highest enum value, but should probably be considered weaker than quorum. We should compare it in some other way.

Maybe you could find something useful in `db/consistency_level.hh`?